Assets/MixedRealityToolkit.Tests/EditModeTests/Core/TestFixture_01_MixedRealityToolkitTests.cs

```diff
@@ -594,10 +595,90 @@ public void Test_07_02_DisableServicesByType()
             }
         }
 
+        #region Multiple Instances Tests
+
+        [Test]
+        public void Test_08_01_CreateMultipleInstances()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene();
+
+            MixedRealityToolkit secondInstance = new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
+            MixedRealityToolkit thirdInstance = new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
+
+            Assert.AreNotEqual(secondInstance, MixedRealityToolkit.Instance);
+            Assert.AreNotEqual(thirdInstance, MixedRealityToolkit.Instance);
+            Assert.IsFalse(secondInstance.IsActiveInstance);
+            Assert.IsFalse(thirdInstance.IsActiveInstance);
+        }
+
+        [Test]
+        public void Test_08_02_SwitchBetweenActiveInstances()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene();
+
+            MixedRealityToolkit secondInstance = new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
+            MixedRealityToolkit thirdInstance = new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
+
+            Assert.AreNotEqual(secondInstance, MixedRealityToolkit.Instance);
+            Assert.AreNotEqual(thirdInstance, MixedRealityToolkit.Instance);
+            Assert.IsFalse(secondInstance.IsActiveInstance);
+            Assert.IsFalse(thirdInstance.IsActiveInstance);
+
+            MixedRealityToolkit.SetActiveInstance(secondInstance);
+
+            Assert.AreEqual(secondInstance, MixedRealityToolkit.Instance);
+            Assert.IsTrue(secondInstance.IsActiveInstance);
+
+            MixedRealityToolkit.SetActiveInstance(thirdInstance);
+
+            Assert.AreEqual(thirdInstance, MixedRealityToolkit.Instance);
+            Assert.IsTrue(thirdInstance.IsActiveInstance);
+        }
+
+        [Test]
+        public void Test_08_03_DestroyActiveInstance()
+        {
+            TestUtilities.InitializeMixedRealityToolkitScene();
```

Comment:
Ooh, this seems like a way that we can actually load / destroy scenes in the playmode tests.